ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -3,7 +3,7 @@
 # local
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
-
+import ivy
```

Comment:
The convention in ivy is to use `import ivy` in tests as little as possible, I guess it's not needed here

Reply:
I am using it because I want to produce ivy.int64 and ivy.int32 specifically in the test case:
```
    seed=helpers.dtype_and_values(
        available_dtypes=tuple([ivy.int64, ivy.int32]),
        min_value=0,
        max_value=10,
        min_num_dims=1,
        max_num_dims=1,
        min_dim_size=2,
        max_dim_size=2
    )
```

I am not sure what other ways I can do without using ivy.int64 & ivy.int32 specifically.

A sidenote:

Tensorflow somehow requires the `shape` parameter has its elements in form of int32 or int64. (not stated in documentation)
`seed` requires to be in form of int32 or int64 as stated in documentation. 

[https://www.tensorflow.org/api_docs/python/tf/random/stateless_uniform](https://www.tensorflow.org/api_docs/python/tf/random/stateless_uniform)

Reply:
That should work, please remove the offending `import ivy` as well